doc/languages-frameworks/qt.section.md

```diff
@@ -2,14 +2,9 @@
 
 Writing Nix expressions for Qt libraries and applications is largely similar as for other C++ software.
 This section assumes some knowledge of the latter.
-There are two problems that the Nixpkgs Qt infrastructure addresses,
-which are not shared by other C++ software:
 
-1.  There are usually multiple supported versions of Qt in Nixpkgs.
-    All of a package's dependencies must be built with the same version of Qt.
-    This is similar to the version constraints imposed on interpreted languages like Python.
-2.  Qt makes extensive use of runtime dependency detection.
-    Runtime dependencies are made into build dependencies through wrappers.
+However, Qt makes extensive use of runtime dependency detection.
+In nixpkgs, those runtime dependencies are made into build dependencies through wrappers.
```

Comment:
If I didn't know already, I would think *made into build dependencies* here means that the build will error out when runtime dependencies are missing during build-time.

Is there a way we can rephrase and signal correctly that this is adds *tracking* for runtime dependencies at build-time?

Reply:
How's this?